cmd/nerdctl/container_restart.go

```diff
@@ -41,51 +37,33 @@ func newRestartCommand() *cobra.Command {
 	return restartCommand
 }
 
-func restartAction(cmd *cobra.Command, args []string) error {
-	// Time to wait after sending a SIGTERM and before sending a SIGKILL.
+func processRestartCommandFlags(cmd *cobra.Command) (types.ContainerRestartOptions, error) {
 	globalOptions, err := processRootCmdFlags(cmd)
 	if err != nil {
-		return err
-	}
-	var timeout *time.Duration
-	if cmd.Flags().Changed("time") {
-		timeValue, err := cmd.Flags().GetUint("time")
-		if err != nil {
-			return err
-		}
-		t := time.Duration(timeValue) * time.Second
-		timeout = &t
```

Comment:
can you keep this logic in `processContainerRestartOptions` and put `Timeout *time.Duration` as a field in `types.ContainerRestartOptions` (instead of `Time uint`)? This is consistent with `ContainerStopOptions`

Reply:
Make sense. Done.